superset/migrations/versions/453530256cea_add_save_option_column_to_db_model.py

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""add_save_form_column_to_db_model
+
+Revision ID: 453530256cea
+Revises: d416d0d715cc
+Create Date: 2021-04-30 10:55:07.009994
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "453530256cea"
+down_revision = "d416d0d715cc"
+
+import sqlalchemy as sa
+from alembic import op
+
+
+def upgrade():
+    with op.batch_alter_table("dbs") as batch_op:
+        batch_op.add_column(
+            sa.Column("save_option", sa.VARCHAR(255), server_default="SQL_ALCHEMY")
```

Comment:
I think `save_option` is not a super informative nam), something like `configuration_method` would be better. And let's use `SQLALCHEMY_URI` as the default.

Reply:
That sounds good, I appreciate you helping me with naming conventions, it is something that I definitely struggle with.